code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1182,11 +1290,19 @@
 								if (ckey(param) == ckey(A.name))
 									M = A
 									break
-						if (M)
 #ifdef TWITCH_BOT_ALLOWED
 							if (IS_TWITCH_CONTROLLED(M))
 								return
 #endif
+						else
+							var/list/target_list = src.get_targets(1, "mob") // Chrunb Erbrbt and Scales To Lizard highfive!
+							if(length(target_list))
+								M = input(src, "Pick someone to high-five!", "EmotiConsole v1.1.3") as null|anything in target_list
```

Comment:
i'd like to see this using the tgui input list

Reply:
Aight, made them use the tgui input list, and it looks like it works pretty good!